src/Microsoft.Graph/Generated/model/Dictionary.cs

```diff
@@ -0,0 +1,37 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+// **NOTE** This file was generated by a tool and any changes will be overwritten.
+// <auto-generated/>
+
+// Template Source: ComplexType.cs.tt
+
+namespace Microsoft.Graph
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// The type Dictionary.
+    /// </summary>
+    [JsonConverter(typeof(DerivedTypeConverter<Dictionary>))]
+    public partial class Dictionary
```

Comment:
Will need to validate whether this will cause collisions with class provided in System libs

Reply:
After testing, looks like this is okay since the system type uses generic parameters and thus no type collision seems to occur.